staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -477,7 +477,8 @@ type JobSpec struct {
 	// The value must be a valid domain-prefixed path (e.g. acme.io/foo) -
 	// all characters before the first "/" must be a valid subdomain as defined
 	// by RFC 1123. All characters trailing the first "/" must be valid HTTP Path
-	// characters as defined by RFC 3986. The value cannot exceed 64 characters.
+	// characters as defined by RFC 3986. The value cannot exceed 63 characters.
+	// The field is immutable.
```

Comment:
nit:

```suggestion
	// This field is immutable.
```

might be better

Reply:
Probably slightly, let me update.

Reply:
Done, while doing so I realized I forgot to update ` pkg/apis/batch/types.go`. Now done.